src/components/FocusTrap/FocusTrapForModal/index.web.tsx

```diff
@@ -9,7 +9,7 @@ function FocusTrapForModal({children, active}: FocusTrapForModalProps) {
             active={active}
             focusTrapOptions={{
                 trapStack: sharedTrapStack,
-                allowOutsideClick: true,
+                clickOutsideDeactivates: true,
```

Comment:
I changed this because in our case clicking outside the modal closes it automatically. With `clickOutsideDeactivate` the focus trap is deactivated instantly and the click behaviour is not modified (eg. input is focused correctly). WIth `allowOutsideClick` focus trap isn't deactivated instantly but focus only temporarily leaves the trap and then returns only to be lost by closing modal.